backend/nestjs/apps/listing-service/src/hotels/hotels.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
Controller must listen to message broker (Kafka) and react to correspond event and doesn't expose apis, following on the architecture